taichi/math/linalg.h

```diff
@@ -510,19 +510,7 @@ struct VectorND : public VectorNDBase<dim__, T, ISE> {
     return ret;
   }
 
-  TI_IO_DECL {
-    if (TI_SERIALIZER_IS(TextSerializer)) {
-      std::string ret = "(";
-      for (int i = 0; i < dim - 1; i++) {
-        ret += fmt::format("{}, ", d[i]);
-      }
-      ret += fmt::format("{}", d[dim - 1]);
-      ret += ")";
-      serializer("vec", ret);
-    } else {
-      TI_IO(d);
-    }
-  }
+  TI_IO_DEF(d);
```

Comment:
As we replace every ser/de with JSON ser/de we no longer need differentiate them.